### PR TITLE
docs: Testkube CLI docker

### PR DESCRIPTION
**Pull request by vLia**
## Pull request description 

Testkube CLI dockerized docs

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/articles/testkube-cli-docker.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Testkube Docker CLI
+
+The easiest way to start managing the tests of your team on a remote Testkube server is to run the Testkube CLI using the official Docker image available starting from Testkube version 1.13. The Testkube CLI Docker image is a self-contained environment that allows you to run Testkube commands in a consistent and isolated manner.
```
Comment (jfermi) [inline docs/docs/articles/testkube-cli-docker.md:3]:
```suggestion
Starting with Testkube version 1.13, the easiest way to start managing your team's tests on a remote Testkube server is to run the Testkube CLI using the official Docker image. The Testkube CLI Docker image is a self-contained environment that allows you to run Testkube commands in a consistent and isolated manner.
```

File: docs/docs/articles/testkube-cli-docker.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The Testkube CLI Docker image is available on Docker Hub. You can pull it using the following command:
+
+```bash
+docker pull testkube/cli:latest
+```
+
+### 2. Building from Source
+
+If you prefer to build the Docker image from source, you can clone the Testkube CLI repository from GitHub and build it locally using goreleaser, the provided Dockerfile and the Makefile. Follow these steps:
```
Comment (jfermi) [inline docs/docs/articles/testkube-cli-docker.md:29]:
```suggestion
If you prefer to build the Docker image from source, you can clone the Testkube CLI repository from GitHub and build it locally using GoReleaser, the provided Dockerfile and the Makefile. Follow these steps:
```

File: docs/docs/articles/testkube-cli-docker.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+2. Build the Docker image:
+
+```bash
+make docker-build-cli DOCKER_BUILDX_CACHE_FROM=type=registry,ref=docker.io/kubeshop/testkube-cli:latest ALPINE_IMAGE=alpine:3.18.0 DOCKER_IMAGE_TAG=local ANALYTICS_TRACKING_ID="" ANALYTICS_API_KEY=""
+```
+
+## Running the Testkube CLI Docker Image
+
+Once you have the image, try running the following command pointing to Testkube demo:
```
Comment (jfermi) [inline docs/docs/articles/testkube-cli-docker.md:45]:
```suggestion
Once you have the image, run the following command pointing to the Testkube demo:
```

File: docs/docs/articles/testkube-cli-docker.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+```bash
+docker run kubeshop/testkube-cli:latest version --namespace testkube --api-uri https://demo.testkube.io/results --client direct
+```
+
+This command starts a new Docker container with the Testkube CLI image and executes the command `testkube version` pointing to the api-server running on the Testkube demo environment.
```
Comment (jfermi) [inline docs/docs/articles/testkube-cli-docker.md:51]:
```suggestion
This command starts a new Docker container with the Testkube CLI image and executes the command `testkube version`, pointing to the api-server running on the Testkube demo environment.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Review by vsukhin — APPROVED** [review]

**Pull request merged**